### PR TITLE
ScanPolarity to wizard and to raw data overview as quick control

### LINKED ISSUE #1836: Polarity filter in wizard
Add "any" option and set filter in chromatogram builder

### PR DESCRIPTION
**Pull request by robinschmid**

### CODE REVIEW

**Comment by robinschmid** [comment]
Done from my side. 
One more thing for another PR would be how to tell people that chromatogram building on both polarities does not work

**Comment by ansgarkorf** [comment]
> Done from my side. One more thing for another PR would be how to tell people that chromatogram building on both polarities does not work

We can validate the parameters before triggering the task and show a message box if polarity "ANY" was chosen for polarity switching data. The message should also point the user to polarity scan filter option

**Comment by robinschmid** [comment]
Not sure its usually multiple tasks running in parallel - so we would need to have a mechanism to only show one dialog. 
Also I would say the dialog should not block the processing but just show up. 

I will create a new issue for this.
#1846

File: mzmine-community/src/main/java/io/github/mzmine/modules/visualization/chromatogramandspectra/ChromatogramPlotControlPane.java [hunk 1]
```diff
@@ -89,7 +95,15 @@ public ChromatogramPlotControlPane(final ParameterSet parameters) {
       }
     });
 
-    HBox controlsWrap = new HBox(5, cbXIC, btnUpdateXIC, btnParam);
+    polarityCombo = new ComboBox<>(FXCollections.observableList(
+        List.of(WizardMsPolarity.No_filter, WizardMsPolarity.Positive, WizardMsPolarity.Negative)));
```
Comment (SteffenHeu) [inline mzmine-community/src/main/java/io/github/mzmine/modules/visualization/chromatogramandspectra/ChromatogramPlotControlPane.java:99]: stick to PolarityType

Comment (SteffenHeu) [reply mzmine-community/src/main/java/io/github/mzmine/modules/visualization/chromatogramandspectra/ChromatogramPlotControlPane.java:99]: use string converter -> any -> no filter

File: mzmine-community/src/main/java/io/github/mzmine/modules/tools/batchwizard/subparameters/custom_parameters/WizardMsPolarity.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ * WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ * OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package io.github.mzmine.modules.tools.batchwizard.subparameters.custom_parameters;
+
+import io.github.mzmine.datamodel.PolarityType;
+
+public enum WizardMsPolarity {
```
Comment (SteffenHeu) [inline mzmine-community/src/main/java/io/github/mzmine/modules/tools/batchwizard/subparameters/custom_parameters/WizardMsPolarity.java:30]: make package private to wizard package

Comment (robinschmid) [reply mzmine-community/src/main/java/io/github/mzmine/modules/tools/batchwizard/subparameters/custom_parameters/WizardMsPolarity.java:30]: problem is that it is being used by the parameters and the builders that are in different subpackages. So cannot be package private

File: mzmine-community/src/main/java/io/github/mzmine/modules/visualization/chromatogramandspectra/ChromatogramPlotControlPane.java (hunk 1 above)
Comment (robinschmid) [reply mzmine-community/src/main/java/io/github/mzmine/modules/visualization/chromatogramandspectra/ChromatogramPlotControlPane.java:99]: using PolarityType now and made a PolarityFilterComboBox to hide the StringConverter No filter is now Any polarity

File: mzmine-community/src/main/java/io/github/mzmine/modules/tools/batchwizard/subparameters/custom_parameters/WizardMsPolarity.java (hunk 2 above)
Comment (robinschmid) [reply mzmine-community/src/main/java/io/github/mzmine/modules/tools/batchwizard/subparameters/custom_parameters/WizardMsPolarity.java:30]: at least now its only used in wizard

**Comment by robinschmid** [comment]
All done

**Pull request merged**